DevSkim-VSCode-Plugin/client/common/comments.json

```diff
@@ -0,0 +1,70 @@
+﻿[
```

Comment:
We eventually want to remove this extra copy of the comments and languages file. We should instead send these over with a custom notification like we do for sending fixes, which can happen each time the configuration is changed (which might change the languages and comments)